src/loader/babel.ts

```diff
@@ -210,20 +210,19 @@ const babelPluginUntyped: PluginItem = function (
 
 export default babelPluginUntyped;
 
-function containsIncompleteCodeblock(line = "") {
-  const codeDelimiters = line
-    .split("\n")
-    .filter((line) => line.startsWith("```")).length;
-  return !!(codeDelimiters % 2);
+function isExampleBlock(line = "") {
+  return line.includes("@example");
```

Comment:
I think we should support both spec ways: triple back ticks and `@example` blocks

Reply:
I re-add this and improve the readability of the function.